ibm/service/vpc/resource_ibm_is_instance.go

```diff
@@ -75,13 +75,15 @@ const (
 	isInstanceDeleteDone              = "done"
 	isInstanceFailed                  = "failed"
 
-	isInstanceStatusRestarting     = "restarting"
-	isInstanceStatusStarting       = "starting"
-	isInstanceActionStatusStopping = "stopping"
-	isInstanceActionStatusStopped  = "stopped"
-	isInstanceStatusPending        = "pending"
-	isInstanceStatusRunning        = "running"
-	isInstanceStatusFailed         = "failed"
+	isInstanceStatusRestarting           = "restarting"
+	isInstanceStatusStarting             = "starting"
+	isInstanceActionStatusStopping       = "stopping"
+	isInstanceActionStatusStopped        = "stopped"
+	isInstanceStatusPending              = "pending"
+	isInstanceStatusRunning              = "running"
+	isInstanceStatusFailed               = "failed"
+	isInstanceAvailablePolicyHostFailure = "availability_policy_host_failure"
+	isInstanceHostFailure                = "host_failure"
```

Comment:
where are we using this isInstanceHostFailure

Reply:
removed